src/js/scriptlets/dom-inspector.js

```diff
@@ -537,9 +538,8 @@ var cosmeticFilterMapper = (function() {
         for ( entry of (details.procedural || []) ) {
             nodes = entry.exec();
             for ( node of nodes ) {
-                if ( filterMap.has(node) === false ) {
-                    filterMap.set(node, entry.raw);
-                }
+                // Upgrade declarative selector to procedural one
+                filterMap.set(node, entry.raw);
```

Comment:
I will need to address this in the future, there used to be a time were I was reporting all filters matching a node. I can't remember why I removed this ability, I probably felt the code was growing complicated and needed to simplify until I got all under control. Anyway, what is the thinking for overriding plain CSS selector with a procedural one when there is collision? (I don't a set opinion, just curious).